source/indexes/compound-index.txt

```diff
@@ -0,0 +1,91 @@
+.. _c-compound-index:
+
+================
+Compound Indexes
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+:manual:`Compound indexes </core/index-compound/>` hold references to multiple
+fields within a collection's documents, improving query and sort performance.
```

Comment:
S: add a sentence like "Use the ``mongoc_collection_create_indexes_with_opts()`` method to create an index.